internal/controller/impersonatorconfig/impersonator_config_test.go

```diff
@@ -376,7 +376,7 @@ func TestImpersonatorConfigControllerSync(t *testing.T) {
 			// This fake server is enough like the real impersonation proxy server for this unit test because it
```

Comment:
This comment still mentions the stopCh (on the line above this one).

Reply:
fixed in next push